sponsorship_switzerland/models/account_invoice.py

```diff
@@ -93,3 +93,15 @@ def group_or_split_reconcile(self):
         """Reconcile given invoices with partner open payments.
         """
         return self._group_or_split_reconcile()
+
+    def action_invoice_open(self):
+        condition = [
+            ("transaction_id", "=", self.transaction_id),
```

Comment:
Beware that in any method, `self` can be a recordset of multiple records (multiple invoices here). Accessing `self.transaction_id` may fail in such cases, so the best is to first loop over the recordset and test each invoice:
```
for invoice in self:
    invoice.transaction_id
```